test/test.js

```diff
@@ -10,6 +10,12 @@ function sortLeftToRight(a, b) {
 	return 0;
 }
 
+function assertTree(tree, nbElements) {
+	assert.strictEqual(tree._getCount(tree.root), nbElements);
+	assert.strictEqual(tree._isSorted(tree.root), true);
```

Comment:
You do know this is not equivalent to the tests of mine that you deleted.  To be equivalent you would have to run this method and check against all my tests, making this kind of pointless.  As far as verifiability of correctness,  this is not as good. You should only be adding tests, not removing them.  Not that doing what you're doing is a bad idea.  It should be in addition to, not replacing.  You have to explicitly check the different cases, otherwise correctness is only based on assumption that isBlanaced and isSorted are correct (there are no tests for them).